### ISSUE TITLE
[News&Press] Arrow next to 'News Articles' is misplaced, when app used on tablets.

### ISSUE THREAD

**Issue by VlasovD** [WorldHealthOrganization/app#1097]
### Steps to reproduce
1. Run app on tablet (iPad or Android tablet)
2. Navigate to News & Press
3. Check 'News Articles'

### What was expected to happen
Arrow next to description should be in line with other arrows on screen.
### What actually happened
Arrow is placed right at the end of description text.

### Device info (if relevant)
* Device type: iPad Mini RETINA, Samsung Galaxy Tab 4
* OS version: iOS 9.0.2, Android 4.4.2
* App version: v0.7.1 (14)

### Screenshots
<details>
<summary>Screenshots</summary>

![IMG_0131](https://user-images.githubusercontent.com/16258539/79989495-96f97080-84b8-11ea-9be3-33b5d51ef5c0.PNG)

![Screenshot_2020-04-22-16-27-29](https://user-images.githubusercontent.com/16258539/79989637-c4461e80-84b8-11ea-986c-d9cb2e8e8cf2.png)
</details>
-->
---

**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the issue).**
- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Issue has descriptive title focused on the feature or problem from the user's perspective

**Comment by sumitnagarro** [comment]
I can work on this.
